### DISCUSSION TITLE
api.dbTable.create inputs [Apr 22]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1933] [Discord Threads]
> 🙋‍♂️ **onlyspike**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Possible to lookup the definition inside the ts ?  
#  
> 💁 **o1lab**  
#### we will improve this  
#  
> 💁 **o1lab**  
####   
![image0](https://cdn.discordapp.com/attachments/966645919372161045/966649087875227699/Screenshot_2022-04-21_at_11.38.06.png)  
#  
> 🙋‍♂️ **onlyspike**  
#### Thank you for your response. I will investigate the ts.  
#  
> 🙋‍♂️ **onlyspike**  
#### It looks like it accept a "data" parameter. I will give this a try and see if it works.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/966645919372161045)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
